packages/web-components/src/popover/popover.definition.ts

```diff
@@ -0,0 +1,9 @@
+import { Popover } from './popover';
+import { popoverTemplate } from './popover.template';
+import { styles } from './popover.styles';
+
+export const definition = Popover.compose({
+  name: 'fluent-popover',
```

Comment:
The `fluent` prefix perhaps should not be hardcoded.

Reply:
Yes, we'll want to setup a "DesignSystem" - https://github.com/chrisdholt/fluentui/blob/features/fast-element-2-fluent-2-reset/packages/web-components/src/fluent-design-system.ts